readthedocs/vcs_support/backends/git.py

```diff
@@ -282,7 +291,9 @@ def are_submodules_available(self, config):
             return False
 
         # Keep compatibility with previous projects
-        return bool(self.submodules)
+        # TODO: remove when all projects are required
+        # to have a config file.
+        return any(self.submodules)
```

Comment:
This results in 2 calls to `git config` for projects that don't have a config file.